pkg/elemental/elemental.go

```diff
@@ -43,7 +43,14 @@ func NewElemental(config *agentConfig.Config) *Elemental {
 
 // FormatPartition will format an already existing partition
 func (e *Elemental) FormatPartition(part *v1.Partition, opts ...string) error {
-	e.config.Logger.Infof("Formatting '%s' partition", part.Name)
+	var name string
```

Comment:
this is just UX, the DeviceMapper partition info is gathered via udev info. For luks devices the name does not appear anywhere so you would get an ugly message for those with no name `Formatting '' partition` 

This does a fallback in case its empty so at least the log message works and user is informed and doesnt look like we screwed up something.